parsec/core/gui/settings_dialog.py

```diff
@@ -1,18 +1,22 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
```

Comment:
I think this whole file can be deleted, it's never imported and `settings_dialog.ui` doesn't exist either.